larreco/TrackFinder/PMAlgTrackMaker_module_continued.h

```diff
@@ -0,0 +1,1039 @@
+void PMAlgTrackMaker::produce(art::Event& evt)
+{
+
+	fDetProp = lar::providerFrom<detinfo::DetectorPropertiesService>();
+
+	reset(evt); // set default values, clear containers at the beginning of each event
```

Comment:
Do you also clear at end of event?  If not you should.  It's probably best to do both beginning and end of event.  At end to reduce peak memory usage and at beginning in case art continued after a throw from within your module ( which may never happend today but may in the future ).